packages/guides-theme-rst/resources/template/rst/inline/emphasis.rst.twig

```diff
@@ -1 +1 @@
-*{{- node.value|raw -}}*
+*{{- node|plaintext -}}*
```

Comment:
When I introduced the inline nested nodes I did use the value because that would not require a change in the templates. As templates can be overwritten by consuming projects. I did see that as a backward compatibility issue. 

This was the reason to try to be backwards compatible and accept the issues in our pipeline. How do you see that? Bc in templates is a very complex problem as I experienced in the past. But maybe I will try to be too strict about this project?

Reply:
I'm not sure I can follow you. The original content still works, but triggers a deprecation. Are you thinking of a case where the user overrides `getValue()` of a node that now has a BC break because it isn't called anymore?

You can use `Deprecation::triggerIfCalledFromOutside()` in `getValue()` then (or the Symfony tactic: at a virtual new boolean parameter to `getValue()` that determines whether the deprecation should be triggered). This should not produce a deprecation when called from a Guides template, but does when called by a user.

In general, deprecations are most useful if all of them can be fixed by a user without upgrading the package triggering the deprecation. If Guides triggers a deprecation from itself, I would never know if I can safely upgrade to 2.0.